mysql-async/src/test/java/com/github/jasync/sql/db/mysql/BitSpec.kt

```diff
@@ -0,0 +1,68 @@
+package com.github.jasync.sql.db.mysql
+
+import org.junit.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+
+class BitSpec : ConnectionHelper() {
+
+
+    @Test
+    fun `result in binary data` () {
+
+      withConnection {
```

Comment:
In tests that use a connection I suggest to do something like:
` fun `result in binary data` () = withConnection { connection -> ...`
It will reduce nesting level and keep the boilerplate out